src/fragments/lib/datastore/ios/relational/save-snippet.mdx

```diff
@@ -1,28 +1,22 @@
 <BlockSwitcher>
 
-<Block name="Listener (iOS 11+)">
+<Block name="Async/Await">
 
 ```swift
-let postWithComments = Post(title: "My post with comments",
-                            rating: 5,
-                            status: .active)
-
-let comment = Comment(content: "Loving Amplify DataStore", post: postWithComments)
-
-Amplify.DataStore.save(postWithComments) { postResult in
-    switch postResult {
-    case .failure(let error):
-        print("Error adding post - \(error.localizedDescription)")
-    case .success:
-        Amplify.DataStore.save(comment) { commentResult in
-            switch commentResult {
-            case .success:
-                print("Comment saved!")
-            case .failure(let error):
-                print("Error adding comment - \(error.localizedDescription)")
-            }
-        }
-    }
+let postWithComments = Post(
+            id: "123", title: "My post with comments", content: "Amplify is Awesome!", 
+            createdAt: .now(),rating: 5, status:.published
+        )
```

Comment:
please follow previous example, ie. parameters line up, close brace not on a separate line, first parameter on same line as the constructor name.